### ISSUE TITLE
Ошибки в консоли при запуске в CentOS 7

### ISSUE THREAD

**Issue by drnextgis** [nextgis/nextgismanager#67]
```
** (wxgiscat:13568): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-cSdhKPGq0j: Connection refused
Gtk-Message: Failed to load module "canberra-gtk-module"

(wxgiscat:13568): Gtk-WARNING **: gtk_menu_attach_to_widget(): menu already attached to GtkMenuItem

(wxgiscat:13568): Gtk-WARNING **: gtk_menu_attach_to_widget(): menu already attached to GtkMenuItem

(wxgiscat:13568): Gtk-WARNING **: gtk_menu_attach_to_widget(): menu already attached to GtkMenuItem

(wxgiscat:13568): Gtk-WARNING **: gtk_menu_attach_to_widget(): menu already attached to GtkMenuItem

(wxgiscat:13568): Gtk-WARNING **: gtk_menu_attach_to_widget(): menu already attached to GtkMenuItem

(wxgiscat:13568): Gtk-WARNING **: gtk_menu_attach_to_widget(): menu already attached to GtkMenuItem

(wxgiscat:13568): Gtk-WARNING **: gtk_menu_attach_to_widget(): menu already attached to GtkMenuItem
```

**Comment by drnextgis** [comment]
Ругань на canberra-gtk-module решается установкой:

```
yum install libcanberra-gtk3
```

**Comment by drnextgis** [comment]
[Bug 1056820 - Every app with a graphic display complains "Couldn't register with accessibility bus"](https://bugzilla.redhat.com/show_bug.cgi?id=1056820) Проблему можно решить так:

```
export NO_AT_BRIDGE=1
```

**Comment by drnextgis** [comment]
Остальные warтing-и не относятся к CentOS, так как проявляются и в других дистрибутивах, поэтому закрываем.
